Healthy/Classes/Extensions/UIView+Style.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+// MARK: - Apply Shadow
+//
+extension UIView {
+    enum ShadowType {
+        case normal
+    }
+
+    struct ShadowStyle {
+        var offset: CGSize
+        var radius: CGFloat
+        var color: CGColor
+        var opacity: Float
+    }
+
+    func applyShadow(_ type: ShadowType) {
+        switch type {
+        case .normal:
+            let offset = CGSize(width: 0, height: 0)
+            let color = UIColor(red: 0.412, green: 0.412, blue: 0.412, alpha: 0.1).cgColor
```

Comment:
Shall we use the colors in the color scheme?

Reply:
Am not sure, the shadow color in Figma design is in hex without a design system color name. What do you think?

Reply:
Choose gray40 ... The closest to this color code